release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -376,6 +376,16 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 [id="ocp-4-10-bare-metal-hardware-bug-fixes"]
 ==== Bare Metal Hardware Provisioning
 
+* Previously, virtual media based deployments of {product-title} have been observed to intermittently fail on iDRAC hardware types. This occurred when outstanding Lifecycle Controller jobs clashed with virtual media configuration requests. With this update, virtual media deployment failure has been fixed by purging any Lifecycle Controller job while registering iDRAC hardware prior to deployment. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1988879[*BZ#1988879*])
+
+* Previously, the `curl` utility used by the machine downloader image did not support classless inter-domain routing (CIDR) with `no_proxy`. A a result, any CIDR added to `noProxy` was ignored when downloading the {op-system-first} image. With this update, proxies are now removed from the environment before calling `curl` when appropriate. As a result, when downloading the machine image, any CIDR in `no_proxy` is no longer ignored. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1990556[*BZ#1990556*])
+
+* Previously, Ironic failed to attach virtual media images for provisioning SuperMicro X11/X12 servers because these models expect a non-standard deice strong (`UsbCd`) for CD-based virtual media. With this update, provisioning now overrides `UsbCd` on SuperMicro machines provisioned with CD-based virtual media. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2009555[*BZ#2009555*])
+
+* Previously, Ironic failed to attach virtual media images on SuperMicro X11/X12 servers due to overly restrictive URl validations on the BMCs of these machines. With this update, the `filename` parameter has now been removed from the URL if the virtual media image is backed by a local file. As a result, the parameter is still passed if the image is backed by an object store. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011626[*BZ#2011626*])
```

Comment:
I am not sure about the last sentence saying "still passed" maybe "still passes" for present tense? I am not 100% on that though.